### ISSUE TITLE
DateInput - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4812]
Add Accessibility information to the DateInput page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the DateInput component and if they are passing/failing.
- [ ] Create a pull request on the Design System site to add this information to the DateInput page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/4675/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [ ] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

**Comment by CillianHPE** [comment]
@jcfilben 

**[Here is the Date Input A11y Audit file](https://hpe-my.sharepoint.com/:w:/p/cillian_pender/EXX40ZdBujdPkpSLSCaXJTsBn-BC3IgfC6e_w3706E6UpA?e=OkySGa)**

**Comment by jcfilben** [comment]
Thanks @CillianHPE, added some comments to the file

**Comment by CillianHPE** [comment]
@jcfilben 

Created an **[issue](https://github.com/grommet/hpe-design-system/issues/4878)** in the backlog for the date input

**Comment by CillianHPE** [comment]
@jcfilben 

Here are the issues created for this work in the Grommet repo
[Date Input - 2.4.3 Focus Order](https://github.com/grommet/grommet/issues/7556)

**Comment by jcfilben** [comment]
Thanks @CillianHPE! Marking this issue as closed
